src/napari/__init__.py

```diff
@@ -14,6 +18,78 @@
 limit_numpy1x_threads_on_macos_arm()
 
 
+# TODO: probably move this elsewhere
+def register_napari_font():
+    import os  # TODO: WTF why do I need this here???
+
+    font_dir = Path(
+        resources.files(__package__).joinpath('resources', 'fonts')
+    )
+
+    if sys.platform == 'darwin':
+        from ctypes import POINTER, c_bool, c_char_p, c_uint32, c_void_p
+
+        from vispy.ext.cocoapy import cf, ct
+
+        # following vispy's implementation in vispy.ext.cocoapy
+        # and vispy.utils.fonts._quartz.py
+        cf.CFURLCreateFromFileSystemRepresentation.restype = c_void_p
+        cf.CFURLCreateFromFileSystemRepresentation.argtypes = [
+            c_void_p,
+            c_char_p,
+            c_uint32,
+            c_bool,
+        ]
+
+        ct.CTFontManagerRegisterFontsForURL.restype = c_bool
+        ct.CTFontManagerRegisterFontsForURL.argtypes = [
+            c_void_p,
+            c_uint32,
+            POINTER(c_void_p),
+        ]
+
+        # vispy/qt use quartz for font discovery on mac
+        for font_file in font_dir.glob('*.ttf'):
```

Comment:
It take me to much time to find first problem. font is in subdirectory of `fonts`
```suggestion
        for font_file in font_dir.glob('*/*.ttf'):
```

Reply:
Aaaahhhhh, damn. Of course :facepalm: Thanks for looking into this, sorry for the silly mistake :sweat_smile: